framer/Screen.coffee

```diff
@@ -24,6 +24,15 @@ class ScreenClass extends BaseClass
 	toInspect: ->
 		return "<Screen #{Utils.roundWhole(@width)}x#{Utils.roundWhole(@height)}>"
 
+	# Point Conversion
+
+	convertPointToLayer: (point, layer) =>
+		return Utils.convertPointFromContext(point, layer, false, true)
+
+	convertPointToCanvas: (point) =>
+		ctx = Framer.CurrentContext.device.context
```

Comment:
Shorter version would be `Framer.Device`. I think the device is always there. If you choose `No Device` from the device picker this still returns a so called `fullscreen` device.